javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -76,6 +76,9 @@ public ProxyMethod map(Operation operation) {
         } else {
             singleValueType = GenericType.BodyResponse(responseBodyType);
         }
+        if (operation.getResponses().stream().anyMatch(r -> Boolean.TRUE.equals(r.getBinary()))) {
+            singleValueType = ClassType.StreamResponse;
+        }
```

Comment:
Might not be a good solution. It will break we get mixed BinaryResponse and SchemaResponse. Any suggestion for improvement?